src/services/lnd/health.ts

```diff
@@ -39,17 +36,19 @@ const isLndUp = async (param: LndParamsUnAuthed): Promise<void> => {
   }
 
   const authParam = authParams.find((p) => p.socket === socket)
-  if (authParam) {
-    authParam.active = active
+  if (!authParam) {
+    throw new Error("unreachable: this should not happen, authParam should not be null")
```

Comment:
not sure about this.. this can crash the service... are we sure to do this?

Reply:
what would you do?

it's really to silence typescript that I have done this.

authParam is typed LndParamsAuthed | undefined after authParams.find((p) => p.socket === socket), but really it can only be LndParamsAuthed (but typescript doesn't understand this).

maybe I can just cast the type instead?